aggregator/elem.go

```diff
@@ -357,20 +364,17 @@ func NewTimerElem(id id.RawID, sp policy.StoragePolicy, aggTypes policy.Aggregat
 	return e
 }
 
-// ResetSetData overrides elemBase.ResetData, to include quantile update for timer elements.
+// ResetSetData resets the timer element and sets data.
 func (e *TimerElem) ResetSetData(id id.RawID, sp policy.StoragePolicy, aggTypes policy.AggregationTypes) {
-	e.elemBase.ResetSetData(id, sp, aggTypes)
-
-	if aggTypes.IsDefault() {
-		// TODO(cw) Remove UseDefaultAggregation in aggregation opts and
-		// store it in elemBase.
-		e.aggOpts.ResetSetData(e.opts.DefaultTimerAggregationTypes())
-		e.isAggTypesPooled = false
+	isDefault := aggTypes.IsDefault()
```

Comment:
Yeah I think `useDefaultAggTypes` is more intuitive, can we replace `isDefault` with `useDefaultAggTypes` wherever applicable?